weather.py

```diff
@@ -0,0 +1,50 @@
+import os
+import datetime
+import requests
+import sqlite3 as lite
+
+# Connect to database
+con = lite.connect('weather.db')
+cur = con.cursor()
+
+# Cities dictionary - key names here are the same as the city column names in the database
+cities = { "new_york": '40.663619,-73.938589',
+            "boston": '42.331960,-71.020173',
+            "nashville": '36.171800,-86.785002',
+            "philadelphia": '40.009376,-75.133346',
+            "washington_dc": '38.904103,-77.017229'
+        }
+
+# Build query - https://api.forecast.io/forecast/APIKEY/LATITUDE,LONGITUDE,TIME
+def get_forecast_data(city, request_time):
+	base = "https://api.forecast.io/forecast/"
+	api_key = os.environ["FORECAST_KEY"] + "/"
```

Comment:
both `base` and `api_key` are constants. by housing them here, you are redefining them every time the function gets run. it's trivial, but sloppy. better to define them at the top of the file. capital letters are preferred. e.g.
```python
BASE = "https://api.forecast.io/forecast/"
API_KEY = os.environ["FORECAST_KEY"] + "/"
```